pylops/signalprocessing/dtcwt.py

```diff
@@ -0,0 +1,147 @@
+__all__ = ["DTCWT"]
+
+from typing import Union
+
+import dtcwt
```

Comment:
this needs to be wrapped in a `pylops.utils.deps` check like here:

https://github.com/PyLops/pylops/blob/dev/pylops/_torchoperator.py#LL5-LL10